data-es2016plus.js

```diff
@@ -3281,28 +3281,191 @@ exports.tests = [
     significance: 'small',
     spec: 'https://github.com/tc39/proposal-regexp-unicode-property-escapes',
     mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Unicode_Property_Escapes',
-    exec: function () {/*
-    const regexGreekSymbol = /\p{Script=Greek}/u;
-    return regexGreekSymbol.test('π');
-  */},
-    res: {
-      babel6corejs2: true,
-      ie11: false,
-      firefox2: false,
-      firefox77: false,
-      firefox78: true,
-      opera10_50: false,
-      chrome59: chrome.harmony,
-      chrome64: true,
-      safari11_1: true,
-      safaritp: true,
-      duktape2_0: false,
-      jerryscript2_3_0: false,
-      graalvm19: true,
-      hermes0_7_0: false,
-      reactnative0_70_3: true,
-      rhino1_7_13: false
-    }
+    subtests: [
+      {
+        name: 'basic',
+        exec: function () {/*
+        const regexGreekSymbol = /\p{Script=Greek}/u;
+        return regexGreekSymbol.test('π');
+      */},
+        res: {
+          babel6corejs2: true,
+          ie11: false,
+          firefox2: false,
+          firefox77: false,
+          firefox78: true,
+          opera10_50: false,
+          chrome59: chrome.harmony,
+          chrome64: true,
+          safari11_1: true,
+          safaritp: true,
+          duktape2_0: false,
+          jerryscript2_3_0: false,
+          graalvm19: true,
+          hermes0_7_0: false,
+          reactnative0_70_3: true,
+          rhino1_7_13: false
+        }
+      },
+      {
+        name: 'Unicode 11',
+        // 2018	June 5
```

Comment:
Added the release date here so if we add new browser in the future, it can help determine the first supported version since any browser released earlier than 2018 June 5 probably does not support Unicode 11, unless they are built against the Unicode beta release.